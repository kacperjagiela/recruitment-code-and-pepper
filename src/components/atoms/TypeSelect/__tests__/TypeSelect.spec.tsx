import { render } from '@testing-library/react';

import { TypeSelect } from '..';

describe('<TypeSelect />', () => {
    it('renders correctly', () => {
        const { container } = render(<TypeSelect options={['a']} onChange={jest.fn} label="test" />);

        expect(container).toMatchSnapshot();
    });
});
