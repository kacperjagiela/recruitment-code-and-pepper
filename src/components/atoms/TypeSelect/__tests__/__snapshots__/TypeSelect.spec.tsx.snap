// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TypeSelect /> renders correctly 1`] = `
<div>
  <div
    class="MuiFormControl-root MuiFormControl-fullWidth css-cq5a0p-MuiFormControl-root"
  >
    <label
      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink="false"
      id="select_input_test"
    >
      test
    </label>
    <div
      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiSelect-root css-tj4fkw-MuiInputBase-root-MuiOutlinedInput-root"
    >
      <div
        aria-expanded="false"
        aria-haspopup="listbox"
        aria-labelledby="select_input_test select_test"
        class="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
        id="select_test"
        role="button"
        tabindex="0"
      >
        <span
          class="notranslate"
        >
          â€‹
        </span>
      </div>
      <input
        aria-hidden="true"
        class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
        tabindex="-1"
        value=""
      />
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
        data-testid="ArrowDropDownIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-dhh0px"
        >
          <span>
            test
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
